### DISCUSSION TITLE
kleiner Funkmodul und Antennen Erfahrungsbericht

### DISCUSSION THREAD

**Discussion by raccoon0815** [tbnobody/OpenDTU#414] [Show and tell]
Ich habe ein paar Test mit Antennen und Modulen durchgeführt 

Funkmodule:
NRF24L01+ von Kuman
NRF24L01+ von DollaTek 
NRF24L01+ bzw. E01-ML01DP5 von EBYTE 

Antennen:
Standard die bei den NRF24L01+ von DollaTek  und Kuman dabei ist 
Nelawya WLAN-Antenne 6 dBi SMA-Stecker Dualband 2,4 GHz 5,8 GHz MIMO-Antenne
Bingfu Dualband WLAN Antenne 2,4/5,8 GHz 8 dBi SMA-Stecker Antenne

Inverter:
HM-800
HM-400

NRF24L01+ von Kuman:
Hier waren von 10 Modulen 5 bei Lieferung defekt und 2 nach einer Woche Betrieb. Die Module können daher nicht wirklich empfohlen werden.

NRF24L01+ von DollaTek 
Durchgehend sauber Leistung und funktionieren einwandfrei. Bei weiterer Entfernung (ca. 40m um zwei  Hausecken und eine Ziegelmauer) konnten nicht immer eine stabile Verbindung hergestellt werden - aber war zum Großteil ok.

NRF24L01+ bzw. E01-ML01DP5 von EBYTE.
Mein Favorit, merklich stabilere Empfangsleistung als die oberen beiden Module (bezogen auf meine Situation)

EBYTE + Antenne von Nelawya bzw. Bingfu:  weiter Verbesserung der Stabilität feststellbar. Bei Neustart der DTU war die Verbindung  nach ca. 10-15 Sekunden da, beim DollaTek mit Standard Antenne waren es um die 60-70 Sekunden.
In Kombination mit dem E01-ML01DP5 von EBYTE  habe ich damit auch Daten im Keller problemlos abfragen können ( 2 Betondecken) 

Es wurde immer mit selben ESP32 getestet.

Bei kurzer Entfernung ohne viel Beton reicht das Standard NRF24L01+ mit Standardantenne locker aus.
Wenn es „verbauter“ wird, dann würde ich auf das EBYTE Modul wechseln ggf. mit größerer Antenne.

**Comment by Nawile** [comment]
Die gute Erfahrung mit dem EBYTE-Modul kann ich bestätigen. Vorher Probleme mit NRF24L01+ von AZ-Delivery (integrierte Antenne) und erhöhtem PA-Level. Mit dem EBYTE und 0815-Antenne keine Probleme mehr bei PA-Level Minimum.

**Comment by themanfrommoon** [comment]
Moin,

ich finde das Thema hochinteressant!
Leider fehlen hier Links zu allen Produkten.
Wie kann man identifizieren, ob man ein NRF24L01+ von Kuman oder ein NRF24L01+ von DollaTek hat?
Welche Antenne von Antenne von Nelawya bzw. Bingfu ist gemeint?

Vielen Dank für die Aufklärung der Fragen!

Beste Grüße,
Chris

**Comment by glucosum** [comment]
Hallo.
Ein sehr schönes Projekt.
Ich habe es auf meine Art gemacht.
Teile habe ich bei Aliexpress bestellt.

https://ae01.alicdn.com/kf/H54d5658e525b4e9f9c80a95dc1a3332eo/NRF24L01-2-4G-wireless-data-transmission-module-2-4GHz-NRF24L01-upgrade-version-NRF24L01-PA-LNA-1000.jpg_640x640.jpg

+

https://ae01.alicdn.com/kf/H5676a219f62a46ddbe046bb197dde676l/ESP32-Development-Board-WiFi-Bluetooth-Ultra-Low-Power-Consumption-Dual-Core-ESP-32-ESP-32S-ESP.jpg_640x640.jpg

+

https://ae01.alicdn.com/kf/He4709f5d82f348f29b1fb98be36924b90/2-4G-antenna-folded-Bluetooth-wifi-module-ZigBee-antenna-SMA-to-IPEX-feeder-line-2-4GHz.jpg_Q90.jpg_. webp

aus diesen Auktionen:

https://www.aliexpress.com/item/1005001621773806.html?spm=a2g0o.order_detail.order_detail_item.19.73e043ceZS0Sdu

https://www.aliexpress.com/item/1005001621991136.html?spm=a2g0o.order_detail.order_detail_item.11.73e043ceZS0Sdu&gatewayAdapt=glo2pol

https://www.aliexpress.com/item/1005001621916223.html?spm=a2g0o.order_detail.order_detail_item.21.73e043ceZS0Sdu

https://www.aliexpress.com/item/1005001621976056.html?spm=a2g0o.order_detail.order_detail_item.7.73e043ceZS0Sdu

Der Effekt sieht so aus:
<a href="https://ibb.co/7nhz29L"><img src="https://i.ibb.co/7nhz29L/IMG20230329104210.jpg" alt="IMG20230329104210" border="0">
<a href="https://ibb.co/CVyr8Ng"><img src="https://i.ibb.co/CVyr8Ng/IMG20230329104143-BURST000-COVER.jpg" alt="IMG20230329104143-BURST000- COVER" border="0"></a>

Ich bereite dafür PCB in KiCad vor.

**Comment by huckes** [comment]
Hi, sehr informative Aufstellung, vielen Dank.
Hatte am Anfang ein Modul ohne Antenne von Hailege verbaut. Entfernung zum Wechselrichter ca. 6m, eine Ziegelwand dazwischen. Hat bestens funktioniert mit niedrigster Sendeleistung (-18dB). Nun kam ein zweiter Wechselrichter dazu, Entfernung ca. 10m, zwei Ziegelwände dazwischen: Kein Empfang, selbst bei höchster Leistung. Dann den Wechselrichter so gedreht, dass die Antenne 90° zu DTU steht, damit geht's jetzt grad so mit höchster Sendeleistung. Dann noch ein Fiyang-Modul mit Antenne aus China an die DTU angeschlossen, ging mal kurz, dann gar nicht.  Hab' mir jetzt das oben empfohlene Ebyte-Modul bestellt.

In dem Zusammenhang wäre es interessant, ob man die Empfangsleistung der NRF-Module ermitteln und anzeigen könnte. Dann wäre auch eine quantitative Aussage zur RF-Leistung der jeweiligen Module möglich.

Beste Grüße, Hans

Update 6.4.23:
Die beiden bestellten EBYTE-Module sind jetzt angekommen und wurden mit vorhandenen WiFi-6mm-Antennen verbaut. Konnte den Sendepegel jetzt wieder auf Minimum herunter setzten und die Kommunikation mit dem entfernteren Wechselrichter ist immer noch besser als mit dem Hailege-Modulen mit den Printantennen und max. Pegel. Das Fiyang-Modul mit Antenne habe ich entsorgt, das hat wohl 'nen Treffer bekommen.

Frohe Ostern, Hans
